common/journal_entries/03_red_scare.txt

```diff
@@ -46,7 +46,8 @@
 							}
 						}
 					}
-					any_country = {
```

Comment:
this is a non-regular event, we can afford to be splurge cpu time here, (and not have weird half year interactions because of slow list updates)

Reply:
wait that would interact badly with the invalid, nvm